BE/src/main/java/kr/codesquad/issuetracker/domain/issue/Issue.java

```diff
@@ -0,0 +1,67 @@
+package kr.codesquad.issuetracker.domain.issue;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import kr.codesquad.issuetracker.domain.BaseTimeEntity;
+import kr.codesquad.issuetracker.domain.comment.Comment;
+import kr.codesquad.issuetracker.domain.label.Label;
+import kr.codesquad.issuetracker.domain.member.Member;
+import kr.codesquad.issuetracker.domain.milestone.Milestone;
+
+@Entity
+@Table(name = "issue")
+public class Issue extends BaseTimeEntity {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name = "issue_id")
+	private Long id;
+
+	private String title;
+
+	@Enumerated(EnumType.STRING)
+	private Status status;
+
+	private String content;
+
+	@ManyToOne(fetch = FetchType.LAZY)
+	@JoinColumn(name = "member_id")
+	private Member member;
```

Comment:
작성자를 명확하게 뜻하는 변수명을 사용하는건 어떨까요?


```suggestion
	private Member writer;
```

Reply:
엔티티에 들어가는 변수명도 테이블과 같은 변수명이아닌 사용할려고하는 이유를 생각해서 지어보도록 하겠습니다 감사합니다 :)